src/docfx/Errors.cs

```diff
@@ -79,6 +79,12 @@ public static Error MicrosoftGraphApiFailed(string exMessage)
 
         public static Error MetadataValidationRuleset(string ruleset, string url)
             => new(ErrorLevel.Info, "MetadataValidationRuleset", $"Metadata validation ruleset used: {ruleset}. Document url: {url}");
+
+        public static Error ContentValidationRuleset(string ruleset, string url)
```

Comment:
Consider use only 1 error code. We can add parameters to the message to let it show up in logs.

Reply:
the html report will use the error code to identify the type, if we use the same error code, which one should they use?

Reply:
The distinct union

Reply:
but the build report call it metadata validation ruleset, so it should only display the `metadata ruleset`

Reply:
Can we make it something like `{type} validation ruleset used: {ruleset}. Document url: {url}`?

Reply:
Or, since reporting side is only a RegEx match, we could change it to whatever string we'd like to have.

Reply:
If these rulesets are not supposed to show up in the report, why log them in the first place?

Reply:
It would help troubleshooting I suppose.

Reply:
If for troubleshooting, writing to the console would suffice

Reply:
@yufeih Because the log is the contract between docfx and op, so we need to log the `metadatavalidationruleset`, off course we can dinstinguish the "metadatavalidationruleset", then only log it and ignore the others(or write the others to console), but is that necessary? what is your concern to log them all?
@herohua the report generator ont only match the message, it use the logcode as well, it sounds confused to me to use the same "MetadataValidationRuleset" for all the three types of validation(metadata, build, content).